faustjs.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
We'll want to add `faustjs.code-workspace` to our `.gitignore` since this is user config.